### PR TITLE
Golang: Covered General Syntax, Arrays, Slice, Maps, Pointers, Struct

### PR DESCRIPTION
**Pull request by riya-singhal31**
Title: Made new branch

This is my Pull Request message, I have made the separate branch for week 2 and have done the recommended corrections.

### CODE REVIEW

**Review by subhamkrai — CHANGES_REQUESTED** [review]
remove unwanted files

**Comment by riya-singhal31** [comment]
> remove unwanted files

Done

**Review by subhamkrai — CHANGES_REQUESTED** [review]
this should not be a new commit. Please remove this in the commit where you added it.

File: Riya Singhal/Golang/generalSyntax.go [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+	if number2 == 31 {
+		fmt.Print("True\n")
+	}
+
+	//for loop
+	for x := 1; x < 5; x++ {
+		fmt.Println(x)
+	}
+
+	//for loop as while loop
```
Comment (Yuggupta27) [inline Riya Singhal/Golang/generalSyntax.go]: Leave an empty space before starting a comment.
`// comment` looks lunch cleaner than `//comment`

Please update this here and everywhere else

**Review by Yuggupta27 — CHANGES_REQUESTED** [review]
Leave an empty space before starting a comment.

**Pull request merged**